mesheryctl/internal/cli/root/system/check.go

```diff
@@ -80,6 +80,14 @@ func NewHealthChecker(options *HealthCheckOptions) (*HealthChecker, error) {
 		return nil, err
 	}
 
+	if utils.PlatformFlag != "" {
+		if utils.PlatformFlag == "docker" || utils.PlatformFlag == "kubernetes" {
+			currCtx.SetPlatform(utils.PlatformFlag)
+		} else {
+			return nil, fmt.Errorf("the platform '%s' is not supported. Supported platforms are:\n\n- docker\n- kubernetes\n\nVerify this setting in your meshconfig at %s or verify by executing `mesheryctl system context view`", utils.PlatformFlag, utils.CfgFile)
```

Comment:
This error could be made to a MeshKit error 🤔

Reply:
ohh yeah, cool!

Reply:
It's same in `start.go` I just checked.

Reply:
Could you change that as well?